Code/Tools/AssetProcessor/native/utilities/PlatformConfiguration.cpp

```diff
@@ -1326,6 +1321,19 @@ namespace AssetProcessor
         }
     }
 
+    int PlatformConfiguration::GetProjectScanFolderOrder() const
+    {
+        auto mainProjectScanFolder = FindScanFolder([](const AssetProcessor::ScanFolderInfo& scanFolderInfo) -> bool
+            {
+                return scanFolderInfo.GetPortableKey() == "Project/Assets";
```

Comment:
Can you make the "Project/Asset" into a `constexpr const char*` constant?

Reply:
Or if you want to get really "meta", make the ProjectScanFolder key a setting itself.

Reply:
I went with a string constant.